db.json

```diff
@@ -6369,6 +6373,10 @@
     "compressible": true,
     "extensions": ["vtt"]
   },
+  "text/x-ansi": {
+    "compressible": true,
+    "extensions": ["ans"]
+  },
```

Comment:
I see someone else remembers the BBS days :) My best (only) references for `text/x-ansi`:
* https://lists.freedesktop.org/archives/xdg/2006-March/006214.html
* https://de.wikipedia.org/wiki/ANSI-Art (references the first)
* https://github.com/eight04/ansi-viewer

For my purposes, I've written a "modern" (Ha!) BBS in Node.js that relies on node-mime, which in turn relies on mime-db for various mappings.